experiments/experiments.go

```diff
@@ -334,3 +337,41 @@ func isSimpleExperiment(experimentType string) bool {
 	}
 	return false
 }
+
+// ExperimentEvent is the playload used by Expose to log whether a user has
+// been exposed to an experimental treatment.
+type ExperimentEvent struct {
+	// ID uniquely identifies the experiment event. If you pass in uuid.Nil the
+	// logger handling this event should generate a UUID v4 (optional).
+	ID uuid.UUID
+	// CorrelationID are used to track events across different services.
+	CorrelationID uuid.UUID
```

Comment:
🔕 Is it that every field didn't mention "(optional)" is required? It doesn't sound right that ID is optional but CorrelationID is required.